File: Exemplos/App_IntentDigitalHUB_XamarinForms/Elgin_IntentDigitalHUB_XamarinForms-font/E1_Digital_Hub_Xamarin_Forms_M8/IntentServices/Bridge/ConsultarStatus.cs
﻿namespace Xamarin_Forms_Intent_Digital_Hub.IntentServices.Bridge
{
    class ConsultarStatus : BridgeCommand
    {
        public ConsultarStatus() : base("ConsultarStatus") { }

        protected override string FunctionParameters()
        {
            return "";
        }
    }
}

File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/BRIDGE/ConsultarStatus.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE

import com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE.BridgeCommand

class ConsultarStatus : BridgeCommand("ConsultarStatus") {
    override fun functionParameters(): String {
        return ""
    }
}